css-cascade/specificity.css

```diff
@@ -0,0 +1,19 @@
+* {
```

Comment:
These selectors should be in the reverse order to demonstrate that specificity can over ride source order!

Reply:
Went ahead and corrected the order.